src/components/Card/Card.js

```diff
@@ -7,8 +7,43 @@ export default class Card extends Component {
       flipped: false,
       unflipped: false
     }
+    this.cardObj = {}
   }
 
+  componentDidMount() {
+    this.cardObj = {
+      name: this.props.cardData.name,
+      type: this.props.cardData.type,
+      main1Label: '',
+      main1: this.props.cardData.main1,
+      main2Label: '',
+      main2: this.props.cardData.main2,
+      secHeader:'',
+      secInfoMain: this.props.cardData.secInfoMain,
+      secInfoOther: this.props.cardData.secInfoOther || '',
+    }
+
+    switch(this.props.cardType) {
```

Comment:
I really like this solution for handling the card type/ data, similar to the switch for the api call, looks good! We'll just need to figure out the weird card CSS thing...